apps/explorer/src/components/NetworkSelector/index.tsx

```diff
@@ -41,25 +36,25 @@ export const NetworkSelector: React.FC<networkSelectorProps> = ({ networkId }) =
     }
   }, [open])
 
-  const redirectToNetwork = (newNetwork: string, currentNetwork: number): void => {
-    const shouldNotRedirectHome = NO_REDIRECT_HOME_ROUTES.some((r: string) => location.pathname.includes(r))
```

Comment:
This logic is important!

It allows users to check their account in multiple networks without needing to search for it again.

Example:

1. Load any account https://explorer.cow.fi/gc/address/0x24b26dba619d4b5d4fd599c5ae69cb95c933ea51
2. Change the network on top
3. It remains in the account page, but for the new network

Any other page should redirect to home, as it makes no sense for example to see the same order/transaction/etc in a different network, but for `/address` it's necessary